tasks/main.yml

```
@@ -0,0 +1,89 @@
+---
+
+- name: Launchpad PPA that provides modern Erlang releases
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: F77F1EDA57EBB1CC
+
+- name: Curl for RabbitMQ
+  shell: curl -1sLf 'https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey' | sudo apt-key add -
+  args:
+    warn: no
+
+- name: Install nessesary package
+  apt: 
+      name: apt-transport-https
+      state: present
+      update_cache: yes
+
+- name: Install basic packages
+  apt:
+    name:
+    - debian-keyring
+    - gnupg
+    - debian-archive-keyring
+    state: latest
```

better change this into `present`

```suggestion
    state: present
```